tests/cgm/test_cgm.py

```diff
@@ -0,0 +1,207 @@
+import pytest
```

Comment:
please add in the test where you take the bg_array and caluclate the percent < 70, percent >=70 and <= 180, and the percent > 180. The test should sum those three percentages and should exactly equal 100%.

Reply:
added test